signalfx.gemspec

```diff
@@ -38,13 +38,12 @@ Gem::Specification.new do |spec|
 
   # protobuf enforces this check but builds with a newer Ruby version so it's not enabled.
   # Incorporating here to allow 2.2.0-1 users to successfully build and install signalfx.
-  active_support_max_version = Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.2.2") ? "<5" : "<6"
+  active_support_max_version = Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.2.2") ? "<5" : "<7"
```

Comment:
I think this will set us up for the issue we're experiencing now and fully reverting would be more favorable:
```ruby
active_support_max_version = "< 5" if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.2.2")
```

Reply:
done